examples/Example5.elm

```diff
@@ -12,7 +13,12 @@ import Html exposing (Html)
 
 main : Program () Model Msg
 main =
-    sandbox { init = init, view = view, update = update }
+    element { init = init, view = view, update = update, subscriptions = subscriptions }
+
+
+subscriptions : Model -> Sub Msg
+subscriptions model =
+    onMouseDown (Dropdown.outsideTarget "my-dropdown" DropdownMsg)
```

Comment:
I was wondering if would be better/clear to instead of exposing the Decoder we expose the whole `Sub Msg` as a "closed box". Maybe I'm missing some other use case that the Decoder can be used, but if that's the only one, we could expose it as a "Plug'n'play" feature.

something like:

```elm
Dropdown.onOutsideDropdownClick "my-dropdown" DropdownMsg
```

Also, another thing that we could change is instead of receiving the id as a String, it can receive the whole dropdown state, this way we guarantee that we will not have typo errors ( we avoid duplicating the id string).

Something like:

```elm
subscriptions : Model -> Sub Msg
subscriptions model =
    onMouseDown (Dropdown.outsideTarget model.dropdownState DropdownMsg)
```

WDYT?

Reply:
This is a great idea, I've combined the two of them in the latest change, thanks for the feedback! 😉